src/Field/Image.php

```diff
@@ -116,6 +122,10 @@ protected function fillThumbnailFields(array $data)
         $size->height = $data['height'];
         $size->mime_type = $data['mime-type'];
 
+        $imgDir = substr($this->url, 0, strrpos($this->url, '/'));
+        $size->url = $imgDir.'/'.$data['file'];
```

Comment:
These two lines weren't intended to be included. I included them by mistake. I have already removed them.